.github/ISSUE_TEMPLATE/update_saas_webiste.md

```diff
@@ -0,0 +1,89 @@
+---
+name: 🏥 SaaS Website Request
+about: Submit a request form for a new Saas client website
+title: ""
+labels: enhancement
+assignees: ""
+---
+
+## What will you update?
+
+<!-- 무엇을 업데이트 할 것 인지 선택해주세요 -->
+<!-- Select what you want to update -->
+<!-- Please skip the this section when you select the 'New' option -->
+
+- [ ] SaaS properties (Primary color, Hospital Name, Template type, etc..)
+- [ ] Domain udpate
+- [ ] Etc
```

Comment:
@sejune-oh 

```
- [ ] Other (please describe):
```

Above wording was taken from `pull_request_template.md`

Reply:
@henrynoowah 

I changed the `Etc` section name to `The Other`
Additionally, when writing `The other`,  I added a request to elaborate at the bottom.